src/routes/adminRoutes.js

```diff
@@ -13,4 +15,9 @@ router.use(
   })
 );
 
+
+router
+  .route('/admin/api/albums')
+  .patch(authenticateUser, authorizePermissions('admin'), adminController.updatePriceOfAlbums);
```

Comment:
Since this admin route is already using the basic auth middleware; we don't need to also add the `authenticateUser` middleware. In addition, both `authenticateUser` and `authorizePermissions` are currently written in a way that's more specific to cookie-based authorization. For now it's sufficient to just rely on the basic auth for these special admin server-rendered pages.

```suggestion
  .patch(adminController.updatePriceOfAlbums);
```